flowsa/data/activitytosectormapping/NAICS_Crosswalk_EPA_CDDPath.csv

```diff
@@ -7,22 +7,19 @@ EPA_CDDPath,Other,NAICS_2012_Code,237130,I
 EPA_CDDPath,Roads and Bridges,NAICS_2012_Code,237310,I
 EPA_CDDPath,Other,NAICS_2012_Code,237990,I
 EPA_CDDPath,Other - Wood,NAICS_2012_Code,237990,I
-EPA_CDDPath,Concrete Processing,NAICS_2012_Code,5629203,
-EPA_CDDPath,Mixed CDD Processing,NAICS_2012_Code,5629202,
-EPA_CDDPath,Aggregate- Drain Rock,NAICS_2012_Code,238910,
-EPA_CDDPath,Aggregate- Other Construction,NAICS_2012_Code,237990,
-EPA_CDDPath,Aggregate- Road Base,NAICS_2012_Code,237310,
+EPA_CDDPath,Aggregate- Drain Rock,NAICS_2012_Code,5629201,
+EPA_CDDPath,Aggregate- Other Construction,NAICS_2012_Code,5629201,
+EPA_CDDPath,Aggregate- Road Base,NAICS_2012_Code,5629201,
 EPA_CDDPath,Landfill,NAICS_2012_Code,5622121,
 EPA_CDDPath,Remanufacture,NAICS_2012_Code,5629201,
-EPA_CDDPath,Remanufacture - Concrete,NAICS_2012_Code,327310,
-EPA_CDDPath,Aggregate,NAICS_2012_Code,212321,
-EPA_CDDPath,Agricultural,NAICS_2012_Code,325314,
-EPA_CDDPath,Compost,NAICS_2012_Code,325314,
-EPA_CDDPath,Compost/Mulch,NAICS_2012_Code,325314,
-EPA_CDDPath,Fill Material,NAICS_2012_Code,212321,
+EPA_CDDPath,Aggregate,NAICS_2012_Code,5629201,
+EPA_CDDPath,Agricultural,NAICS_2012_Code,,
+EPA_CDDPath,Compost,NAICS_2012_Code,5622192,
+EPA_CDDPath,Compost/Mulch,NAICS_2012_Code,5622192,
+EPA_CDDPath,Fill Material,NAICS_2012_Code,,
```

Comment:
Assuming the missing sectors for agricultural and fill material are intentional?

Reply:
I don't know that these are waste management pathways that would be relevant for the waste_national_model. I can map them still I suppose much like you did for Facts and Figures. But what is `fill material`? It is a pathway for `Fines`. @WesIngwersen  any ideas?